app/knight.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import annotations
+
+
+class Knight:
+    def __init__(self, knight: dict) -> None:
+        self.knight = knight

```

Comment:
The `knight` parameter of the `Knight` class constructor is not descriptive enough. Consider renaming it to something more meaningful, such as `knight_attributes` or `knight_properties`. This is a code style issue.